src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const table = document.querySelector('table');
+  const headers = table.querySelectorAll('th');
+  const rows = Array.from(table.querySelectorAll('tbody tr'));
+
+  const sortRows = (index) => {
+    const isNumeric = index === 2 || index === 3;
```

Comment:
The `isNumeric` check assumes that columns at index 2 and 3 are always numeric. Ensure that these indices are correct for your table structure. If the table structure changes, this hardcoded logic may lead to incorrect sorting.